test/app_test_case.py

```diff
@@ -16,6 +25,37 @@
 from azul_test_case import AzulTestCase
 
 log = logging.getLogger(__name__)
+key_dir = os.path.dirname(__file__)
+public_key_path = os.path.abspath(os.path.join(key_dir, 'public.pem'))
+private_key_path = os.path.abspath(os.path.join(key_dir, 'private.pem'))
+
+
+def generate_test_keys():
```

Comment:
This should be a method or property in the test case. The property should generate the keypair if necessary and return the path to it.

Reply:
Currently, this is part of a helper class, called `TestKeyManager` in `app_test_case.py`. I am thinking of either leaving it (to keep it isolated from growing `LocalAppTestCase`) as it is, or merging it into `LocalAppTestCase`. Which approach would you prefer, @hannes-ucsc ?